src/backend/web/templates/index/index_buildseason.html

```diff
@@ -34,18 +34,12 @@ <h2>
         <a class="btn btn-default" href="https://www.firstinspires.org/robotics/frc/game-and-season" target="_blank" rel="noopener noreferrer"><span class="glyphicon glyphicon-info-sign"></span> <i>FIRST</i> 2023 Game and Season Resources</a>
       </div>
 
-      {% if build_handler_show_ri3d != 'False' %}
+      {% if build_handler_show_ri3d %}
       <p>
         <a class="btn btn-lg btn-success btn-block" href="/watch/ri3d"><span class="glyphicon glyphicon-play"></span> Watch Ri3D Live</a>
       </p>
       {% endif %}
 
-      {% if build_handler_show_teameventsearch %}
```

Comment:
This looks like it's been removed from the codebase, so I'm going to remove it while I'm here (a little out of scope)